css/portal-dashboard/answers/answer-modal.less

```diff
@@ -0,0 +1,132 @@
+@import "../variables";
+
+.modal-dialog .modal-content {
+  border-radius: 8px !important;
+}
+
+.lightbox {
+  width: 640px;
+  border-radius: 8px !important;
```

Comment:
Unneeded?

Reply:
Took off the !important. But still need the border-radius.